src/Exception/Api/ValidationException.php

```diff
@@ -11,7 +11,7 @@ class ValidationException extends ClientErrorException
 	/**
 	 * @param mixed[] $fields
 	 */
-	public function __construct(string $message = '', int $code = 422, ?Throwable $previous = null, array $fields = [])
+	public function __construct(string $message = 'Validation error. See details in the context.', int $code = 422, ?Throwable $previous = null, array $fields = [])
```

Comment:
Keep it in PR, but in separate commit please.

Reply:
Sorry for that :(